websub-service.toml

```diff
@@ -70,7 +70,7 @@ INTENT_VERIFICATION_BACKOFF_FACTOR = 2.0
 INTENT_VERIFICATION_MAX_INTERVAL = 20.0
 
 # The maximum retry count
-KAFKA_CONSUMER_MAX_POLL_RECORDS = 50
+KAFKA_CONSUMER_MAX_POLL_RECORDS = 3
```

Comment:
Lets update MESSAGE_DELIVERY_TIMEOUT also to 30.0

Reply:
Done.